osidb/models.py

```diff
@@ -1188,7 +1188,24 @@ def create_tracker(affect, external_system_id, _type, **extra_fields):
 
     def get_queryset(self):
         """define base queryset for retrieving trackers"""
-        return super().get_queryset()
+        return (
+            super()
+            .get_queryset()
+            .annotate(
+                # annotate queryset with embargoed pseudo-attribute as it is fully based on the ACLs
+                embargoed=models.Case(
+                    models.When(
+                        acl_read=[
+                            uuid.UUID(acl)
+                            for acl in generate_acls([settings.EMBARGO_READ_GROUP])
+                        ],
+                        then=True,
+                    ),
+                    default=False,
+                    output_field=models.BooleanField(),
+                )
+            )
+        )
```

Comment:
As a general rule of thumb we try not to add stuff in one commit then remove it in another

Reply:
makes sense, fixed.